index.bs

```diff
@@ -2364,11 +2369,13 @@ When this method is invoked, the user agent MUST execute the following algorithm
 
                 1. [=list/For each=] |credentialMetadata| of |collectedDiscoveredCredentialMetadata|:
 
-                    1. [=map/Set=] |silentlyDiscoveredCredentials|[|credentialMetadata|] to |authenticator|.
+                    1. If |credentialIdFilter| [=list/is empty=] or |credentialIdFilter| contains an item whose
+                        {{PublicKeyCredentialDescriptor/id}}'s value is set to |credentialMetadata|'s [=DiscoverableCredentialMetadata/id=],
```

Comment:
I think this could read slightly better this way:
```suggestion
                        {{PublicKeyCredentialDescriptor/id}}'s value equals <code>|credentialMetadata|.[=DiscoverableCredentialMetadata/id=]</code>,
```